actors/sunbird-lms-mw/actors/common/src/main/java/org/sunbird/user/profile/ProfileUtil.java

```diff
@@ -8,12 +8,13 @@
 public class ProfileUtil {
 
     public static final String OSID = "osid";
+    public static final ObjectMapper mapper = new ObjectMapper();
```

Comment:
I guess we can create a new ObjectMember in JsonSchemaValidator. That would clear objectMapper once the schema is parsed by GC. 
But this way, the ObjectMapper will be in memory till we stop the instance.